docs/getting-started/cli-getting-started.md

```diff
@@ -24,43 +47,85 @@ To install the core Zowe CLI:
 npm install -g @zowe/cli@zowe-v3-lts
 ```
 
-### Installing CLI plug-ins
+If an `EACCESS` error displays and you are running MacOS or Linux systems, there are workarounds available. See the instructions for using the `sudo` command in [Installing Zowe CLI and Zowe CLI plug-ins](../user-guide/cli-installcli).
+
+### Installing Zowe CLI plug-ins
 
 To install Zowe CLI plug-ins:
 
 ```
 zowe plugins install @zowe/cics-for-zowe-cli@zowe-v3-lts @zowe/db2-for-zowe-cli@zowe-v3-lts @zowe/mq-for-zowe-cli@zowe-v3-lts @zowe/zos-ftp-for-zowe-cli@zowe-v3-lts
 ```
 
-The preceding command installs most open-source plug-ins, but the IBM Db2 plug-in requires [additional configuration to install](../user-guide/cli-db2plugin.md#installing).
+The preceding command installs [IBM® CICS® Plug-in for Zowe CLI](../user-guide/cli-cicsplugin), [IBM® Db2® Plug-in for Zowe CLI](../user-guide/cli-db2plugin), [IBM z/OS FTP Plug-in for Zowe CLI](../user-guide/cli-ftpplugin), and [IBM MQ Plug-in for Zowe CLI](../user-guide/cli-mqplugin). Remove a plug-in from the command to not install it.
 
 For more information, see [Installing Zowe CLI plug-ins](../user-guide/cli-installplugins.md).
 
+   :::note
+
+   The IBM Db2 plug-in requires [additional configuration to install](../user-guide/cli-db2plugin.md#installing).
+
+   :::
+
 ## Issuing your first commands
 
 Issue `zowe --help` to display full command help. Append `--help` (alias `-h`) to any command to see available command actions and options.
 
 *Optionally*, you can view the Zowe CLI web help in a browser window. For more information, see [Displaying help](../user-guide/cli-using-displaying-help.md). 
 
-All Zowe CLI commands start with `zowe` followed by the name of the [core command group](../user-guide/cli-using-understanding-core-command-groups.md). For example, `zowe plugins -help`. To interact with the mainframe, type `zowe` followed by a command group, action, and object. Use options to specify your connection details such as password and system name.
+All Zowe CLI commands start with `zowe` followed by the name of the [core command group](../user-guide/cli-using-understanding-core-command-groups.md). For example, `zowe plugins --help`. 
+
+To interact with the mainframe, type `zowe` followed by a command group, action, and object. Use options to specify your connection details such as password and system name.
```

Comment:
Is "object" standard terminology that we use elsewhere? It sounds kind of weird to me but I don't immediately have a better idea 🙂